src/custom/state/claim/hooks/index.ts

```diff
@@ -17,19 +17,40 @@ import { calculateGasMargin } from 'utils/calculateGasMargin'
 import { isAddress } from 'utils'
 
 import { getClaimKey, getClaimsRepoPath, transformRepoClaimsToUserClaims } from 'state/claim/hooks/utils'
+import { registerOnWindow } from 'utils/misc'
+import mockData, { MOCK_INDICES } from './mocks/claimData'
+import { getIndexes } from './utils'
 
 export { useUserClaimData } from '@src/state/claim/hooks'
 
 const CLAIMS_REPO_BRANCH = 'main'
 export const CLAIMS_REPO = `https://raw.githubusercontent.com/gnosis/cow-merkle-drop/${CLAIMS_REPO_BRANCH}/`
 
 export const enum ClaimType {
-  Airdrop = 'Airdrop', // free, no vesting, can be available on both mainnet and gchain
-  GnoOption = 'GnoOption', // paid, with vesting, must use GNO, can be available on both mainnet and gchain
-  UserOption = 'UserOption', // paid, with vesting, must use Native currency, can be available on both mainnet and gchain
-  Investor = 'Investor', // paid, with vesting, must use USDC, only on mainnet
-  Team = 'Team', // free, with vesting, only on mainnet
-  Advisor = 'Advisor', // free, with vesting, only on mainnet
+  Airdrop, // free, no vesting, can be available on both mainnet and gchain
+  GnoOption, // paid, with vesting, must use GNO, can be available on both mainnet and gchain
+  UserOption, // paid, with vesting, must use Native currency, can be available on both mainnet and gchain
+  Investor, // paid, with vesting, must use USDC, only on mainnet
+  Team, // free, with vesting, only on mainnet
+  Advisor, // free, with vesting, only on mainnet
+}
+
+export type TypeToPriceMapper = Map<ClaimType, number>
+
+// Hardcoded values
+export const ClaimTypePriceMap: TypeToPriceMapper = new Map([
+  [ClaimType.GnoOption, 16.66],
+  [ClaimType.Investor, 26.66],
+  [ClaimType.UserOption, 36.66],
+])
+
+export const NamedClaimTypeMap = {
+  [ClaimType.Airdrop]: 'Airdrop',
+  [ClaimType.GnoOption]: 'GnoOption',
+  [ClaimType.UserOption]: 'UserOption',
+  [ClaimType.Investor]: 'Investor',
+  [ClaimType.Team]: 'Team',
+  [ClaimType.Advisor]: 'Advisor',
 }
```

Comment:
I don't think this is needed. `ClaimType[0] === 'Airdrop'`, no?
Same way as `ClaimType['Airdrop'] === 0`

In short, enum (not const enum) already have the bi-directional mapping

Reply:
yep you're right! legend